packages/cicero-core/lib/template.ne

```diff
@@ -1,7 +1,4 @@
 # Dynamically Generated
-@builtin "number.ne"
```

Comment:
Moving the nearley builtin grammars inline avoids a branch in nearley that fails in the browser.